com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1925,16 +1868,7 @@ private void OnClientBeginSync(uint sceneEventId)
                 sceneLoad = SceneManagerHandler.LoadSceneAsync(sceneName, loadSceneMode, sceneEventProgress);
 
                 // Notify local client that a scene load has begun
-                OnSceneEvent?.Invoke(new SceneEvent()
-                {
-                    AsyncOperation = sceneLoad,
-                    SceneEventType = SceneEventType.Load,
-                    LoadSceneMode = loadSceneMode,
-                    SceneName = sceneName,
-                    ClientId = NetworkManager.LocalClientId,
-                });
-
-                OnLoad?.Invoke(NetworkManager.LocalClientId, sceneName, loadSceneMode, sceneLoad);
+                InvokeSceneEvents(NetworkManager.LocalClientId, sceneEventData, sceneLoad);
```

Comment:
Does this need to add the scene path or is that being automatically done?

Reply:
The scenePath is being calculated from the sceneHash inside of `InvokeSceneEvents`